alert-common/src/test/java/com/synopsys/integration/alert/common/util/UrlUtilsTest.java

```diff
@@ -0,0 +1,47 @@
+package com.synopsys.integration.alert.common.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class UrlUtilsTest {
+    @Test
+    public void argWithTrailingSlashTest() {
+        String url = "https://awebsite.com";
+        String formattedUrl = UrlUtils.appendTrailingSlashIfNoneExists(url + "/");
+        assertEquals(url + "/", formattedUrl);
+    }
+
+    @Test
+    public void argWithNoTrailingSlashTest() {
+        String url = "https://awebsite.com";
+        String formattedUrl = UrlUtils.appendTrailingSlashIfNoneExists(url);
+        assertEquals(url + "/", formattedUrl);
+    }
+
+    @Test
+    public void argSurroundedByWhitespaceTest() {
+        String url = "https://awebsite.com/";
+        String formattedUrl = UrlUtils.appendTrailingSlashIfNoneExists("  " + url + "     ");
+        assertEquals(url, formattedUrl);
+    }
+
+    @Test
+    public void nullArgTest() {
+        String formattedUrl = UrlUtils.appendTrailingSlashIfNoneExists(null);
+        assertEquals("", formattedUrl);
```

Comment:
Do we want to use parameterized tests for this stuff or no? I remember a previous PR where parameterized tests were advocated for in this case.

Reply:
I wanted to use them, but we don't seem to have access to that dependency here for some reason. Definitely worth more investigation.